assets/prototype/domain/shared/predicates/tickets/filters/percentSoldAtOrAbove/index.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { is } from 'ramda';
+
+/**
+ * Internal dependencies
+ */
+import { Ticket } from '../../../../../eventEditor/data/types';
+import validFiniteQuantity from '../validFiniteQuantity';
+
+type PercentSoldAtOrAboveProps = {
+	maxQuantity: number;
+	tickets: Ticket[];
+};
+
+/**
+ * @param {Object} ticket event ticket object
+ * @param {number} maxQuantity
+ * @return {boolean} true if sold/qty >= maxQuantity
+ */
+const percentSoldAtOrAbove = ({ maxQuantity, tickets }: PercentSoldAtOrAboveProps) => {
```

Comment:
Not your fault since this was originally my code, but I'm thinking something like `percentage` might be more clear than `maxQuantity` for the variable name, since that's not really a quantity.
I got confused by the tests when I saw `maxQuantity` as an integer like 75 or 90 and was thinking "75 tickets? 90 tickets?" whereas if i had named that var `percentage`, then it could not be so easily confused.

What do you think?

Reply:
Agree, `percentage` is a more accurate description.
https://github.com/eventespresso/event-espresso-core/pull/2150/commits/2fbe27ca53d52995cb7d2c3a59d45a5a9c882027